packages/metrics/src/index.ts

```diff
@@ -62,27 +66,38 @@ class Metrics {
   }
 
   bootstrapSentry() {
-    const { context, ignoreErrors, sentryKey, sentryProject, userID } = this.settings;
-
-    if (!sentryKey || !sentryProject) {
+    const {
+      context,
+      ignoreErrors,
+      sentryDSN,
+      sentryKey,
+      sentryProject,
+      userID,
+      onSentryScope,
+    } = this.settings;
+    const { host, protocol } = global.location;
+
+    if (!sentryDSN && (!sentryKey || !sentryProject)) {
       return;
     }
 
-    const ravenHost = `${global.location.protocol}//${sentryKey}@${global.location.host}`;
-    const ravenPath = `/proxy/sentry/${sentryProject}`;
-
-    Raven.config(`${ravenHost}${ravenPath}`, {
+    initSentry({
+      dsn: sentryDSN || `${protocol}//${sentryKey}@${host}/${sentryProject}`,
+      enabled: true,
+      environment: process.env.NODE_ENV,
       ignoreErrors,
       release: process.env.SENTRY_RELEASE,
-      environment: process.env.NODE_ENV,
-    })
-      .setUserContext({
-        browserLocale: global.navigator.language,
-        userAgent: global.navigator.userAgent,
-        userID: userID || 'N/A',
-        ...context,
-      })
-      .install();
+    });
+
+    configureScope(scope => {
```

Comment:
might be an extra PR, but we update the sentry breadcrumbs with the tracking names as well, to make them more useful.
https://git.musta.ch/airbnb/torch/blob/master/src/setup/setupMetrics.js#L20